### PR TITLE
Display birth date in patient search

### LINKED ISSUE #2182: Show BirthDate in PatientSearch
## Is your feature request related to a problem? Please describe 👀
Request to show birthdate in patient search, just display it for now. Will need to include as a search param in future.

### Moneyworks Jobcode 🧰
OMS:MVP

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #2182 

# 👩🏻‍💻 What does this PR do? 
Displays birthdate in patient search modal. 

# 🧪 How has/should this change been tested? 
- [ ] Create new prescription 
- [ ] See patient's DoB if they have one.

### CODE REVIEW

File: client/packages/system/src/Patient/Components/PatientOptionRenderer/PatientOptionRenderer.tsx [hunk 1]
```diff
@@ -8,23 +8,25 @@ import {
...
+            sx={{
+              whiteSpace: 'nowrap',
+            }}
+          >
+            {item.code}
+          </Typography>
         </Box>
-      </DefaultAutocompleteItemOption>
-    );
+        <Box display="flex" flexDirection="row" gap={1} width={80}>
```
Comment (mark-prins) [inline client/packages/system/src/Patient/Components/PatientOptionRenderer/PatientOptionRenderer.tsx:26]: on my screen, the 80px width isn't enough:

Screenshot 2023-09-11 at 9 14 33 PM

the rows where the dob is `30` do have the full date of birth, but it's wrapping over two rows. Either stretching the width or adding `whitespace: nowrap` would fix

File: client/packages/system/src/Patient/Components/PatientOptionRenderer/PatientOptionRenderer.tsx [hunk 2]
```diff
@@ -8,23 +8,25 @@ import {
...
-              textOverflow="ellipsis"
-              sx={{
-                whiteSpace: 'nowrap',
-              }}
-            >
-              {item.code}
-            </Typography>
-          </Box>
-          <Typography>{item.name}</Typography>
+  (): AutocompleteOptionRenderer<SearchInputPatient> => (props, item) => (
```
Comment (mark-prins) [inline client/packages/system/src/Patient/Components/PatientOptionRenderer/PatientOptionRenderer.tsx:11]: side note: `patient` would read better than `item` 😉

**Review by mark-prins — APPROVED** [review]
👍 

I do wonder if another colour or some other indicator would improve the view though:

Screenshot 2023-09-11 at 9 24 46 PM

and what's with the height of the autocomplete??

**Comment by roxy-dao** [comment]
> 👍
> 
> I do wonder if another colour or some other indicator would improve the view though:
> 
> Screenshot 2023-09-11 at 9 24 46 PM
> and what's with the height of the autocomplete??

Hmm... I haven't done anything to the height 🤔 It's the same for the other AutoComplete components too... is it too much whitespace?

File: client/packages/system/src/Patient/Components/PatientOptionRenderer/PatientOptionRenderer.tsx [hunk 3]
```diff
@@ -20,13 +20,15 @@ export const getPatientOptionRenderer =
...
             }}
           >
-            {item.code}
+            {patient.code}
           </Typography>
         </Box>
-        <Box display="flex" flexDirection="row" gap={1} width={80}>
-          <Typography>{item.dateOfBirth}</Typography>
+        <Box display="flex" flexDirection="row" gap={1} width={100}>
+          <Typography whiteSpace={'nowrap'} color={'secondary.main'}>
```
Comment (mark-prins) [inline client/packages/system/src/Patient/Components/PatientOptionRenderer/PatientOptionRenderer.tsx:27]: just as an aside - there's no need to change, this is more informational. 
you can just specify `secondary` for the colour here 

```suggestion
          <Typography whiteSpace={'nowrap'} color="secondary"}>
```

**Pull request merged**